### PR TITLE
Fix docstring comment for constants

### PR DESCRIPTION
**Pull request by nicochatzi**
As a first contribution, here's a small and possibly quite naive fix and test for #1995  :)

### CODE REVIEW

File: src/codegen/mod.rs [hunk 1]
```diff
@@ -618,6 +618,10 @@ impl CodeGenerator for Var {
             return;
         }
 
+        if let Some(comment) = item.comment(ctx) {
+            result.push(attributes::doc(comment));
+        }
```
Comment (emilio) [inline src/codegen/mod.rs:623]: There's some places here were we could still bail out without generating the constant. Instead, let's move the `let mut attrs = vec![];` from below to here, and append to `attrs`.

With that fixup, this looks great, thanks!

**Review by emilio — CHANGES_REQUESTED** [review]
just one minor change to handle the edge cases. With that fixed it looks great, thanks!

**Review by emilio — APPROVED** [review]
Looks good, thanks!

**Comment by nicochatzi** [comment]
Cool, thanks @emilio!

**Pull request merged**